examples/sites/src/views/components/components/api-docs.vue

```
@@ -0,0 +1,219 @@
+}
+
+const apiExpandConf = {
+  expandAll: false,
+  trigger: 'row',
+  expandRowKeys: [],
+  accordion: false,
+  activeMethod: (row) => row.typeAnchorName,
+  showIcon: true // 配置是否显示展开图标
+}
+
+const emit = defineEmits(['jump-to-demo'])
```

ESLint

[error] defineEmits should be the first statement in `<script setup>` (after any potential import statements or type definitions).

(vue/define-macros-order)